### ISSUE TITLE
Abstand der Suchbegriffe im Titel berücksichtigen

### ISSUE THREAD

**Issue by annekauf** [StabiBerlin/sbb-relevance-test#20]
Suche über alle Felder: Reihenfolge der Treffer korreliert mit Nähe der Suchbegriffe im Titel.
Je näher im Titel die Suchbegriffe beisammen sind, desto weiter oben wird der Treffer angezeigt.

Nach einem Treffer, bei denen zwischen den Suchbegriffen 1 Wort steht, folgt kein Treffer, bei dem zwischen den Suchbegriffen 0 Worte stehen usw.

**Comment by annekauf** [comment]
found this, to be considered / discussed:
pf2 = title^0,02 und ps2 = 2
– Zusatzbewertung für Phrasen mit Wortabstand

**Comment by duncdrum** [comment]
yup lets have a look together, is there a particular search example that shows the improvement?
